lib/index.ts

```diff
@@ -50,9 +58,9 @@ export type IntegrationOptions = {
    */
   apiOptions?: ISbConfig;
   /**
-   * A boolean to enable/disable the Storyblok JavaScript Bridge. Enabled by default.
+   * A boolean to enable/disable the Storyblok JavaScript Bridge or a StoryblokBridgeConfigV2 configuration object. Enabled by default.
```

Comment:
Maybe a bit clearer: ... or _provide_ a StoryblokBridgeConfigV2 ...
What do you think?

Reply:
Sounds good. Updated!